JavaScripit/index.js

```diff
@@ -4,15 +4,6 @@ var footer = document.querySelector("footer");
 var copyright = document.createElement('p');
 footer.appendChild(copyright);
 copyright.innerHTML = "\u00A9 " + thisYear;
-var skills = [
-    "HTML",
-    "CSS",
-    "JavaScript",
-    "Version control(Git)",
-    "Browser Developer Tools",
-    "Debugging and Troubleshooting",
-    "Responsive web Design"
-];
```

Comment:
I would keep this in the JS, if only because it's part of the grading rubric.

(If you were to get rid of it, you'd want to clean up the other unused JS too.